modules/js/perf/perf_imgproc/perf_cvtcolor.js

```diff
@@ -0,0 +1,534 @@
+// you may need install benchmark with npm by 
+// `npm i --save benchmark`
+
+let Benchmark = require('benchmark');
+let cv = require('../../opencv');
+let help_func = require('../perf_helpfunc');
+
+cv.onRuntimeInitialized = () => {
+  console.log('opencv.js loaded');
+  // Benchmark.options.minSamples = 50;     // set the minimum of Samples for each case
+  let suite = new Benchmark.Suite;
+  global.cv = cv;
+  global.help_func = help_func;
+  global.Params = [];
+  let totalTestNum = 0;
+  require('../base');  // need global cv
```

Comment:
i would suggest to expose `cvSize` in `base` module instead of populating `global` inside module loading.